cmd/stunstamp/stunstamp.go

```diff
@@ -534,7 +578,12 @@ func remoteWriteTimeSeries(client *remoteWriteClient, tsCh chan []prompb.TimeSer
 }
 
 func main() {
+	flag.Var(&flagDstPorts, "dst-port", "destination ports to monitor; may be specified multiple times")
 	flag.Parse()
+	if len(flagDstPorts) < 1 {
+		log.Fatal("at least one dst-port flag must be specified")
+	}
+	flagDstPorts = slices.Compact(flagDstPorts)
```

Comment:
I guess this is meant to deduplicate? But it won't do that unless the slice is sorted (e.g., []int{1, 1, 3, 3, 1} → []int{1, 3, 1}).

(If the flag setter takes a string, it could also sort the values so that this would work)